chunker/scripts/runBenchmarkTest.sh

```diff
@@ -17,7 +17,7 @@ for JAR in `ls $LIB`; do
 done
 
 
-TESTDATA="src/main/resources/test.txt"
+TESTDATA="/shared/corpora/corporaWeb/written/eng/chunking/conll2000distributions/test.txt"
```

Comment:
Yeah. The reason I'm not worried about it is that I think some day we will add a new ChunkerTester which takes a test file name as input; after all, we want more people using our code.